pkgs/tools/misc/seaborn-data/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, newScope, fetchFromGitHub, unzip, stdenvNoCC }:
+let
+  base = {
+    version = "unstable-2023-09-16";
```

Comment:
sorry I wasn't very clear there. The commit you are refererencing is from "2023-01-26"
(the ref you are using).

Reply:
Oh it definitely makes more sense, thank you! Fixed!